docs/purposes_yaml_files.md

```diff
@@ -143,20 +141,6 @@ in Sequencescape. Usually only true for the first plate in the pipeline.
 
 Default: `false`
 
-#### :input_plate
-
-**(plate only)**
-Boolean, indicates that the plate has the input_plate flag set
-in Sequencescape. Usually only true for the first plate in the pipeline. Also
-used to determine if the plate shows in the 'New Input Plates' inbox, and to
-determine which barcode gets shown on downstream ancestors
-
-```yaml
-:input_plate: false
-```
-
-Default: `false`
```

Comment:
Is this default implicit?  I don't see any changes to the code that uses these `.yml` config files.

Reply:
the purpose yml files themselves are used by a rake task `config:generate`